onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -943,15 +943,10 @@ TEST(InferenceSessionTests, TestOptionalInputs) {
   status = RunOptionalInputTest(true, true, false);
   ASSERT_TRUE(status.IsOK()) << status.ErrorMessage();
```

Comment:
>ASSERT_TRUE(status.IsOK()) << status.ErrorMessage(); [](start = 2, length = 52)

build failure is from here, since it return an INVALID_ARGUMENT status now.